crates/noirc_frontend/src/ast/expression.rs

```diff
@@ -661,9 +663,7 @@ impl FunctionDefinition {
 
 impl Display for FunctionDefinition {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        if let Some(attribute) = &self.attribute {
-            writeln!(f, "{attribute}")?;
-        }
+        writeln!(f, "{:?}", self.attributes)?;
```

Comment:
Note this is now using debug display, because vectors. This is okay, just noting it